client/signup/steps/plans/index.jsx

```diff
@@ -76,86 +74,6 @@ export class PlansStep extends Component {
 		this.unsubscribe();
 	}
 
-	onSelectPlan = ( cartItem ) => {
-		const {
-			additionalStepData,
-			stepSectionName,
-			stepName,
-			flowName,
-			themeSlugWithRepo,
-			launchSite,
-			goToNextStep,
-		} = this.props;
-
-		if ( cartItem ) {
-			this.props.recordTracksEvent( 'calypso_signup_plan_select', {
-				product_slug: cartItem.product_slug,
-				free_trial: cartItem.free_trial,
```

Comment:
This line is missing in the newly extracted `buildUpgradeFunction()`. Are we sure about removing it?

Reply:
Afaik there aren't any free trials floating around, but I added it back in just in case!